<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-IC1TTNK" name="org.abc.automation.testcases.TC_001_ValidateLoginFunctionality" tests="1" failures="0" timestamp="9 Sep 2020 15:02:24 GMT" time="0.000" errors="1">
  <testcase name="tc_001_login_functionality" time="0.000" classname="org.abc.automation.testcases.TC_001_ValidateLoginFunctionality">
    <error type="org.testng.TestNGException" message="
Method public void org.abc.automation.testcases.TC_001_ValidateLoginFunctionality.tc_001_login_functionality(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : Static">
      <![CDATA[org.testng.TestNGException: 
Method public void org.abc.automation.testcases.TC_001_ValidateLoginFunctionality.tc_001_login_functionality(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : Static
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:263)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:425)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1253)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- tc_001_login_functionality -->
</testsuite> <!-- org.abc.automation.testcases.TC_001_ValidateLoginFunctionality -->
